.github/workflows/continuous-build.yml

```diff
@@ -65,3 +65,6 @@ jobs:
           remote-build-cache-proxy-enabled: false
           # TODO(anuraaga): Remove version specifier after next release creates a tag on master.
           arguments: artifactoryPublish -Prelease.version=0.10.0-SNAPSHOT
+        env:
+          BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
+          BINTRAY_KEY: ${{secrets.BINTRAY_KEY }}
```

Comment:
does the variance in whitespace make any difference here?

Reply:
Which whitespace? Are you referring to the difference in the length of the word USER vs KEY? :P

Reply:
```suggestion
          BINTRAY_KEY: ${{ secrets.BINTRAY_KEY }}
```

Reply:
Thanks my eyes just could not find that (no coffee yet is a problem)